src/main/java/com/commercetools/sync/products/utils/ProductVariantUpdateActionUtils.java

```diff
@@ -40,6 +40,9 @@
         + "setAttribute/setAttributeInAllVariants update action for the attribute with the name '%s' in the "
         + "ProductVariantDraft with key '%s' on the product with key '%s'. Reason: %s";
     private static final String NULL_PRODUCT_VARIANT_ATTRIBUTE = "AttributeDraft is null.";
+    private static final String IMAGE_LISTS_NOT_THE_SAME_SIZE = "Old and new image lists must have the same size, but "
+        + "they have %d and %d respectively";
+    private static final String OLD_IMAGE_NOT_FOUND = "Old image [%s] not found in the new images list.";
```

Comment:
I added the error messages as constants here.

Reply:
i'd not do this: 
  - the messages are not reused, thus no benefit to carry out them
  - they have substitution wildcards (`%s`, `%d`) and it is hard now to read how the substitutions will be done, because the string an actual values are separated

Reply:
agreed. d17e9de